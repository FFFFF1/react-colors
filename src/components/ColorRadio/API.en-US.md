| Property       | Description           | Type             | Default       |
|------------|----------------|------------------|--------------|
| model      | toggle model | enum: 'rgb.r' 'rgb.g' 'rgb.b' 'hsv.h' 'hsv.s' 'hsv.v' | hsv.h | 

