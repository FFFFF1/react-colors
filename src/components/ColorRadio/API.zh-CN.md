| 参数       | 说明           | 类型             | 默认值       |
|------------|----------------|------------------|--------------|
| model      | toggle model | 选一: 'rgb.r' 'rgb.g' 'rgb.b' 'hsv.h' 'hsv.s' 'hsv.v' | hsv.h | 