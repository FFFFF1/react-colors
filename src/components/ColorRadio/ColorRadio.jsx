import React from 'react'

function ColorRadio ({model, ...props}) {
	return <input type='radio' {...props} />
}

ColorRadio.displayName='ColorRadio'

export default ColorRadio