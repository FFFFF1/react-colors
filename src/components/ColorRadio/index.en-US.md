`ColorPanel` is a radio type input, can toggle color space