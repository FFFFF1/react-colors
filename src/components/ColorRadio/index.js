export {default} from './ColorRadio.jsx'