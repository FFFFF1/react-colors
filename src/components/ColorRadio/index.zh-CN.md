`ColorRadio` 是 `radio` 类型的 `input`，用于切换色彩空间。

