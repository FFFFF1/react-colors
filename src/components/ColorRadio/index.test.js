import React from 'react'
import ColorRadio from './ColorRadio.jsx'
import { mount, render } from 'enzyme'
import { renderToJson } from 'enzyme-to-json'

describe('ColorRadio correctly', () => {
	it('render correctly', () => {
		const wrapper = render(<ColorRadio model='rgb.r'/>)
		expect(renderToJson(wrapper)).toMatchSnapshot()
	})
})