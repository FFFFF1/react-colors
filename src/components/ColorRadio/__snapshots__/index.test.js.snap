// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorRadio correctly render correctly 1`] = `
<input
  type="radio"
/>
`;
